Dribbblish/color.ini

```diff
@@ -71,6 +71,24 @@ notification       = bd93f9
 notification-error = e22134
 misc               = BFBFBF
 
+[dracula-purple]

```

Comment:
Add the preview also to THEMES.md and the theme README.md

Reply:
Preview added to the specified locations